linux_os/guide/system/network/network-nftables/set_nftables_loopback_traffic/rule.yml

```diff
@@ -0,0 +1,43 @@
+documentation_complete: true
+
+title: 'Set nftables configuration for loopback traffic'
```

Comment:
In alignment to Style Guide, task names must be in [Title case](https://en.wikipedia.org/wiki/Title_case)